index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300&family=Roboto:wght@100&display=swap"
+      rel="stylesheet">
+
+  </head>
+
+  <body>
+    <div class="container">
+      <header>
+        <nav>
+          <a href="#" id="welcome">Welcome</a>
+          <a href="#"><img src="images/Union.svg" alt=""></a>
+        </nav>
+      </header>
+      <section class="content">
+        <div class="introduction">
+          <h1 id="name">Hey there. I'm Denis</h1>
+          <h1 id="profession">I'm a software developer</h1>
+        </div>
```

Comment:
- [OPTIONAL] Using semantic HTML is not allowed to have to h1 so kindly change that. Keep coding 💪